xmipp

```diff
@@ -1459,31 +1653,37 @@ def ensureConfig():
         sys.exit(-6)
 
 
-def ensureGit():
-    if not checkProgram('git'):
-        print(red('Git not found'))
-        return False
+def ensureGit(critical=False):
+    if not checkProgram('git', critical):
+        if critical or RELEASE_HASH == '*******':
```

Comment:
I would prefer to have `'*******'` as a constant

Reply:
Done